### ISSUE TITLE
DropDown doesn't open when having showOnFocus=false

### ISSUE THREAD

**Issue by DarkSide666** [atk4/ui#580]
There is some issue with dropdown field. It doesn't open when you click on down arrow with mouse. When you start typing something in field, then it opens
field is added in form like this:
```
$format = $g->addField('format', ['DropDown', [
            'isValueRequired' => true,
            'values' => [
                'csv' => ['CSV', 'icon' => 'table icon'],
                'dc' => ['Docurec', 'icon' => 'table icon'],
                'ss' => ['Select System', 'icon' => 'table icon'],
            ],
            'width' => 'three', // otherwise menu options are wrapped
            'dropdownOptions' => ['showOnFocus' => false], // https://github.com/atk4/ui/issues/512
        ]])->set('csv'); // default value
```

Issue is because I have `showOnFocus=false`. I needed this because this field is first in form and it automatically gets focus and expands (when not needed).

**Comment by ibelar** [comment]
@DarkSide666 - This is an issue with Fomantic-ui dropdown. Will setup an issue on their repo.

**Comment by ibelar** [comment]
https://github.com/fomantic/Fomantic-UI/issues/191

**Comment by lubber-de** [comment]
Thanks for the Bug-Report.
Should be fixed by https://github.com/fomantic/Fomantic-UI/pull/193

**Comment by DarkSide666** [comment]
now waiting for next Fomantic release.

**Comment by DarkSide666** [comment]
Fixed by upgrading to Fomantic 2.6.4. version.
